tests/test_jupyterlite_component.py

```diff
@@ -9,7 +9,7 @@
 
 def test_run(tmp_path):
     port = int("80" + str(random.randrange(10, 99)))
-    jupyter = JupyterLite(port=port, contents=str(tmp_path))
+    jupyter = JupyterLite(port=port, host="0.0.0.0", contents=str(tmp_path))
```

Comment:
I think if you do `localhost` windows will pass: I tried it here: https://github.com/Lightning-AI/LAI-JupyterLite-Component/pull/4